src/pages/index.js

```diff
@@ -2,51 +2,90 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import popupWithConfirmation from "../components/popupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import {
-  initialCards,
   settings,
   galleryDisplay,
   nameInput,
   titleInput,
   imgUrlInput,
+  initialCards,
   descriptionInput,
   imgModalForm,
   editButton,
   addImgButton,
   profileForm,
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms